src/dotnet/Program.cs

```diff
@@ -121,22 +121,47 @@ private static int ProcessArgs(string[] args)
                 ["test"] = TestCommand.Run
             };
 
+            int exitCode = 100;
+            
+            string arguments = string.Empty;
+
+
             Func<string[], int> builtIn;
             if (builtIns.TryGetValue(command, out builtIn))
             {
-                return builtIn(appArgs.ToArray());
+                exitCode = builtIn(appArgs.ToArray());
+                
+                appArgs.ToList().ForEach(a => { arguments += a + " "; });
+
+            }
+            else
+            {
+                CommandResult result = Command.Create("dotnet-" + command, appArgs, FrameworkConstants.CommonFrameworks.NetStandardApp15)
```

Comment:
The example says

> We only use var when it's obvious what the variable type is (i.e. var stream = new FileStream(...) not var stream = OpenStandardInput()).

Even `Command.Create().Execute().GetResult()` isn't obvious what type this returns (especially to someone new to the code).

The reason I point this out is that the more all our repos have their own rules, the more annoying things like this are.  Here, someone new to the repo tries to contribute and they get pinged for things like "use var". While other 'dotnet' repos explicitly have rules **against** using var in these situations. Again, having a consistent set of coding-style guidelines across the 'dotnet' repos eliminates these issues.